db/db.go

```diff
@@ -88,55 +92,63 @@ func (db *DB) Query(query string) (RowResults, error) {
 		}
 		results = append(results, r)
 	}
-	log.Debug(func() string { return "Executed query successfully: " + query })
+	log.Debugf("Executed query successfully: %s", query)
 	return results, nil
 }
 
 // Execute executes the given sqlite statement, of a type that doesn't return rows.
 func (db *DB) Execute(stmt string) error {
+	var s string
+
 	_, err := db.dbConn.Exec(stmt)
-	log.Debug(func() string {
```

Comment:
If I recall correctly, this was done for a reason so that the strings are not actually created if logging is not in debug mode.